jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenRawConfiguration.java

```diff
@@ -204,4 +204,11 @@ public Path getImageIdOutputPath() {
   public Path getImageJsonOutputPath() {
     return jibPluginConfiguration.getImageJsonOutputPath();
   }
+
+  @SuppressWarnings("unchecked")
+  @Override
+  public List<ExtensionConfiguration> getPluginExtensions() {
+    return (List<ExtensionConfiguration>)
```

Comment:
This line seems a little crazy to me.

Should we just be using `<? extends ExtensionConfiguration>` everywhere?